micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/svgindex.js

```
@@ -1772,33 +1772,33 @@ const PaymentIcon = ({ className = "", fill = "none", style = {} }) => (
-
-const WarningIcon =({ className = "", fill = "none", style = {} })=>(<svg width="40" height="40" viewBox="0 0 40 40" fill={fill } className={className} style={style}  xmlns="http://www.w3.org/2000/svg">
-<g clip-path="url(#clip0_7513_43945)">
-<path d="M1.66797 35.0007H38.3346L20.0013 3.33398L1.66797 35.0007ZM21.668 30.0007H18.3346V26.6673H21.668V30.0007ZM21.668 23.334H18.3346V16.6673H21.668V23.334Z" fill={COLOR_FILL}/>
-</g>
-<defs>
-<clipPath id="clip0_7513_43945">
-<rect width="40" height="40" fill="white"/>
-</clipPath>
-</defs>
+const HistoryIcon = ({ className = "", fill = "none", style = {} }) => (
+  <svg width="36" height="36" viewBox="0 0 36 36" fill={fill} className={className} style={style} xmlns="http://www.w3.org/2000/svg">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)